.github/workflows/build.yml

```diff
@@ -27,43 +27,39 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  get_ci_fixes:
+  build:
     runs-on: ubuntu-latest
+    container: ghcr.io/sagemath/sage/sage-${{ github.event.inputs.platform || 'ubuntu-focal-standard' }}-with-targets:${{ github.event.inputs.docker_tag || 'dev'}}
     steps:
+      - name: Install git (macos)
+        if: runner.os == 'macOS'
+        run: |
+          brew update
+          brew install git
+
+      - name: Install git (linux)
+        if: runner.os == 'Linux'
+        run: |
+          apt update
+          apt install -y git
```

Comment:
No, one cannot assume that on Linux, `apt` is the package manger.

Reply:
Well, it's working, so I call it good enough...

Reply:
When the workflow is run manually, the container can be selected by the user. As you know.
"That it's working" is because you didn't test it on a system that does not use apt.